cve_bin_tool/checkers/python.py

```diff
@@ -68,9 +68,6 @@ def get_version(lines, filename):
     elif guess_contains_python(lines):
         version_info["is_or_contains"] = "contains"
 
-    elif "python" in lines:
-        version_info["is_or_contains"] = "is"
-
```

Comment:
This is a good change.  I think there's a lot of packages called python-* or *-python depending on your distribution

e.g. https://packages.ubuntu.com/search?keywords=python&searchon=names&suite=eoan&section=all